### PR TITLE
BRITECHSRM262 - Imap -> Email

### PR DESCRIPTION
**Pull request by pallenSL**
Start again.  From git pull.

### CODE REVIEW

File: Email/Email.fsproj [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <ProjectGuid>{2210DA00-A0D3-4035-B7E6-6787AF279A5A}</ProjectGuid>
     <OutputType>Exe</OutputType>
     <RootNamespace>Imap</RootNamespace>
```
Comment (jebbinSCL) [inline Email/Email.fsproj]: Missed the rootNamespace here.

Comment (pallenSL) [inline Email/Email.fsproj]: So much for Find in Files!

Comment (jebbinSCL) [inline Email/Email.fsproj]: What did you use for it? VS? I think if you find in certain setups it may ignore some files?

Comment (pallenSL) [inline Email/Email.fsproj]: Yes, VS ignores the project file.  
$ egrep -i imap FTW!

**Comment by jebbinSCL** [comment]
:shipit:

**Pull request merged**